### DISCUSSION TITLE
Unable to send or receive messages on chat

### DISCUSSION THREAD

**Discussion by vineethbv** [vinsdragonis/Drakespeare#50] [Q&A] [answered]
The messages sent over chat aren't able to access the server at the moment. The console throws an error saying access denied.

**Comment by vinsdragonis** [comment, marked as answer]
> The messages sent over chat aren't able to access the server at the moment. The console throws an error saying access is denied.

This is due to Heroku disabling its free services as many users began to misuse them. We are shortly migrating the server, so that should fix this issue.
